server/cmwell-data-tools/src/main/scala/cmwell/tools/data/ingester/Ingester.scala

```diff
@@ -128,9 +128,12 @@ object Ingester extends DataToolsLogging with DataToolsConfig {
           (IngestFailEvent(numInfotons = infotons.size), stpMetadata)
 
         case (Success(res @ HttpResponse(s, h, e, p)), infotons, stpMetadata) if s.isFailure() =>
-          logger.error(s"${getLabel(stpMetadata)} problem: host=${getHostname(h)} $e, $p")
-          badDataLogger.info(s"${getLabel(stpMetadata)} data: ${infotons.map(_.utf8String).mkString("\n")}")
-          res.discardEntityBytes()
+          e.toStrict(1.minute).map { strict =>
+            logger.error(s"${getLabel(stpMetadata)} problem: host=${getHostname(h)} ${strict.data}, $p")
+            badDataLogger.info(s"${getLabel(stpMetadata)} data: ${infotons.map(_.utf8String).mkString("\n")}")
+            res.discardEntityBytes()
```

Comment:
There is no reason to discard the entity bytes if you already converted it to strict.